core/src/test/java/com/linecorp/armeria/server/ServiceBindingTest.java

```diff
@@ -42,6 +47,27 @@
 
     private static CountDownLatch accessLogWriterCheckLatch;
     private static CountDownLatch propertyCheckLatch;
+    private static final Queue<Integer> decoratorLog = new ArrayBlockingQueue<>(100);
```

Comment:
nit: `LinkedTransferQueue`, which is unbounded? (Just in case we are blocked indefinitely)

Reply:
My reading of the docs is that `add` should throw `IllegalStateException` if we exceed the capacity?

Reply:
Ah, I was rather concerned about consuming side, but yeah, I guess it doesn't matter anyway. Maybe I just find the magic number `100` somewhat unnatural.

Reply:
It's certainly arbitrary. My thinking was that it would be better to hit an exception from hitting the limit than to grow without bound and then OOM. (In practice I doubt either would happen though).